contracts/payloads/IGP62/PayloadIGP62.sol

```diff
@@ -0,0 +1,157 @@
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(PROPOSER_AVO_MULTISIG_3) == PROPOSER_AVO_MULTISIG_3,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Reduce limits old INST-ETH Dex Pool
+        action1();
+
+        // Action 2: Update cbBTC-wBTC dex pool min and max center price
+        action2();
+    }
+
+    function verifyProposal() external view {}
+
+
+    /**
+     * |
+     * |     Team Multisig Actions      |
+     * |__________________________________
+     */
+    function setState(
+        uint256 inst_eth_dex_id,
+        uint256 inst_eth_vault_id,
+        uint256 eth_usdc_dex_id,
+        uint256 eth_usdc_vault_id
+    ) external {
+        if (msg.sender != TEAM_MULTISIG) {
+            revert("not-team-multisig");
+        }
+
+        INST_ETH_DEX_ID = inst_eth_dex_id;
+        INST_ETH_VAULT_ID = inst_eth_vault_id;
+        ETH_USDC_DEX_ID = eth_usdc_dex_id;
+        ETH_USDC_VAULT_ID = eth_usdc_vault_id;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Set fGHO lending rewards
+    function action1() internal {
+        address[] memory protocols = new address[](1);
+        address[] memory tokens = new address[](1);
+        uint256[] memory amounts = new uint256[](1);
+
+        {
+            /// fUSDC
+            IFTokenAdmin(F_GHO_ADDRESS).updateRewards(
+                0x512Ac5b6Cf04f042486A198eDB3c28C6F2c6285A
+            );
+
+            uint256 allowance = IERC20(GHO_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_GHO_ADDRESS
+            );
+
+            protocols[0] = F_GHO_ADDRESS;
+            tokens[0] = GHO_ADDRESS;
+            amounts[0] = allowance + (70_000 * 1e18);
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
```

Comment:
we need to also call start() in this IGP

Reply:
Okay!